### PR TITLE
py: varchar(n) type tests for count aggregate

### PR DESCRIPTION
**Pull request by rivudhk**

### CODE REVIEW

File: python/tests/aggregate_tests/test_varcharn_count.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
+from .aggtst_base import TstView
+
+
+class aggtst_atbl_varcharn(TstView):
+    def __init__(self):
+        # Validated on Postgres
+        self.data = [
+            {"id": 0, "f_c1": None, "f_c2": "abc  "},
```
Comment (mihaibudiu) [inline python/tests/aggregate_tests/test_varcharn_count.py:8]: so postgres also has spaces here?

Comment (rivudhk) [reply python/tests/aggregate_tests/test_varcharn_count.py:8]: Yes. They both have a single space after "c".

Comment (mihaibudiu) [reply python/tests/aggregate_tests/test_varcharn_count.py:8]: I see two spaces

Comment (rivudhk) [reply python/tests/aggregate_tests/test_varcharn_count.py:8]: Yes. Sorry. They both have two spaces after "c". 

This is the result from Postgres. 

"null"	"abc  "
"hello"	"fred"
"@abc"	"varia"
"hello"	"examp"

**Pull request merged**